tests/expressions/tests.py

```diff
@@ -496,7 +496,10 @@ def test_ticket_16731_startswith_lookup(self):
         )
         self.assertSequenceEqual(qs, [e2, e3])
 
-    def test_ticket_18375_join_reuse(self):
+    def test_f_expression_join_reuse(self):
+        """
+        F() works across multijoin relations, join reuse (#18375).
+        """
```

Comment:
```suggestion
```
ticket number exists in the comment underneath

Reply:
same as the above consistency.